tests/actions/encounterTypes.test.js

```diff
@@ -0,0 +1,158 @@
+import expect from 'expect';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import moxios from 'moxios';
```

Comment:
All the import above are globally exported in the setup file. No need to import them again